{"name": "获取同步根组id", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <bms.testcases.test_02_usersync.TestUserSync object at 0x000002197C3CB910>\ncaseid = 'usersync_get_syncgroupid_01'\ncasedata = {'base_url': '${config(host)}', 'caseid': 'usersync_get_syncgroupid_01', 'description': '获取同步根组id', 'epic': '用户管理', ...}\n\n    @pytest.mark.parametrize('caseid, casedata', [(caseid, casedata) for caseid, casedata in\n                                                  readcase.ReadCase().get_case_dict(\n                                                      ROOT_DIR + 'bms/data/usersync/usersync_get_syncgroupid.yaml').items()])\n    def test_usersync_get_syncgroupid(self, caseid, casedata):\n        handleallure.allure_display(casedata)\n        logging.info(\"==========开始执行用例：%s==========\", caseid)\n>       runcase.excute_case(casedata)\n\nbms\\testcases\\test_02_usersync.py:165: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\ncommon\\runcase.py:54: in excute_case\n    checkresult.check_result(hope_result, recv_data, recv_code)\ncommon\\checkresult.py:30: in check_result\n    assert_response(value, real_res)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nhope_res = {'msg': 'success', 'statusCode': 0}\nreal_res = {'msg': '找不到对应的组织机构', 'statusCode': 1114886}\n\n    def assert_response(hope_res, real_res):\n        \"\"\"\n        返回结果校验 -全返回校验\n        :param hope_res:\n        :param real_res:\n        :return:\n        \"\"\"\n        try:\n            with allure.step(\"返回值校验\"):\n                allure.attach(name=\"期望返回值\", body=str(hope_res))\n                allure.attach(name='实际返回值', body=str(real_res))\n                if isinstance(hope_res, dict) and isinstance(real_res, dict):\n                    flag = handledict.cmp_dict(hope_res, real_res)\n>                   assert flag\nE                   AssertionError\n\ncommon\\checkresult.py:45: AssertionError"}, "description": "获取同步根组id", "steps": [{"name": "执行关联接口", "status": "passed", "steps": [{"name": "执行当前接口：同步任务列表搜索-无搜索条件", "status": "passed", "attachments": [{"name": "请求方法：", "source": "f35cf321-8fda-4ceb-84c0-6e5525e12f0d-attachment.attach"}, {"name": "请求地址", "source": "c86362de-7ddd-4f93-ac57-463b90335db7-attachment.attach"}, {"name": "请求头", "source": "f6574b2c-5f46-4e7b-b11d-9518a2c05dfa-attachment.attach"}, {"name": "请求参数", "source": "c8090669-51aa-48f9-ab01-52720e0de827-attachment.attach"}, {"name": "请求结果", "source": "3203ba24-c1ce-4b63-8e3c-e495a7ce0605-attachment.attach"}], "start": 1687260104534, "stop": 1687260104640}, {"name": "结果校验", "status": "passed", "steps": [{"name": "状态码校验", "status": "passed", "attachments": [{"name": "期望状态码", "source": "a333b243-8251-4fe2-a117-f47f9683cca8-attachment.attach"}, {"name": "实际状态码", "source": "b1ac5255-d6ec-41a9-af55-3f974f98b6cd-attachment.attach"}], "start": 1687260104640, "stop": 1687260104640}, {"name": "返回值校验", "status": "passed", "attachments": [{"name": "期望返回值", "source": "0d1599e2-6de6-4bed-b400-b5fd070ef6ce-attachment.attach"}, {"name": "实际返回值", "source": "9d855a21-8490-4390-bc2a-bd22b800170a-attachment.attach"}], "start": 1687260104640, "stop": 1687260104641}], "start": 1687260104640, "stop": 1687260104641}], "start": 1687260104503, "stop": 1687260104643}, {"name": "执行当前接口：获取同步根组id", "status": "passed", "attachments": [{"name": "请求方法：", "source": "e53ad26f-8059-4772-9d29-176c28ee77e2-attachment.attach"}, {"name": "请求地址", "source": "58c18f4f-334d-49da-b15f-f6de04060983-attachment.attach"}, {"name": "请求头", "source": "6cf189b9-75c8-4beb-9f7c-9e6e1486f7e4-attachment.attach"}, {"name": "请求参数", "source": "73ce4097-13d4-4f24-8d3b-91168d01f71d-attachment.attach"}, {"name": "请求结果", "source": "34802907-7a7d-45fd-b772-5a174831e3a8-attachment.attach"}], "start": 1687260104645, "stop": 1687260104739}, {"name": "结果校验", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"E:\\Wondersoft_ApiTest\\common\\checkresult.py\", line 30, in check_result\n    assert_response(value, real_res)\n  File \"E:\\Wondersoft_ApiTest\\common\\checkresult.py\", line 45, in assert_response\n    assert flag\n"}, "steps": [{"name": "状态码校验", "status": "passed", "attachments": [{"name": "期望状态码", "source": "3b11222e-9c45-4702-95fa-8105eaea7c1f-attachment.attach"}, {"name": "实际状态码", "source": "d2015251-4b79-4dd2-a73d-502bf959e247-attachment.attach"}], "start": 1687260104739, "stop": 1687260104742}, {"name": "返回值校验", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"E:\\Wondersoft_ApiTest\\common\\checkresult.py\", line 45, in assert_response\n    assert flag\n"}, "attachments": [{"name": "期望返回值", "source": "14e31cb3-6248-4d52-b976-77e49ba67612-attachment.attach"}, {"name": "实际返回值", "source": "810becec-8389-41ed-ac9b-8564d4970816-attachment.attach"}], "start": 1687260104742, "stop": 1687260104743}], "start": 1687260104739, "stop": 1687260104743}], "attachments": [{"name": "log", "source": "5a31b11b-0d01-4250-b33e-26d14c54012a-attachment.txt", "type": "text/plain"}], "start": 1687260104501, "stop": 1687260104744, "uuid": "f7f556b8-b79b-471f-8f9e-58edd490c20d", "historyId": "c076d0bb657b7fadcb6dc7f1415a3243", "testCaseId": "8a41af8eca098c4295d5e872d129839c", "fullName": "bms.testcases.test_02_usersync.TestUserSync#test_usersync_get_syncgroupid", "labels": [{"name": "epic", "value": "用户管理"}, {"name": "feature", "value": "用户同步管理-用户同步"}, {"name": "story", "value": "编辑，获取同步根组id接口"}, {"name": "parentSuite", "value": "bms.testcases"}, {"name": "suite", "value": "test_02_usersync"}, {"name": "subSuite", "value": "TestUserSync"}, {"name": "host", "value": "OFYI7JX9DURTDV6"}, {"name": "thread", "value": "7756-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "bms.testcases.test_02_usersync"}]}